DANS-KNAW/easy-sword2-dans-examples
# Extended examples to include a sequence of related bags

No JIRA issue

#### When applied it will
* Add examples that can be used to test sending updates of already archived bags.

#### How can this be testen
Testing this can be done by first sending `revision01.zip`, then adding its UUID as a urn:uuid in the Is-Version-Of field of revision02's and revision03's `bag-info.txt`'s  (and updating the bags with bagit update), then sending revision02.zip and revision03.zip. Verify that the bags created in `/var/local/bag-store` for revision02 and revision03 contain the appropriate `fetch.txt`'s .
